openedx/frontend-app-authoring
# feat: preview library block changes in course unit [FC-0062]

## Description

Implements: https://github.com/openedx/frontend-app-authoring/issues/1464

Depends on: 
* https://github.com/openedx/frontend-app-authoring/pull/1422
* https://github.com/openedx/edx-platform/pull/35861

## Supporting information

* https://github.com/openedx/frontend-app-authoring/issues/1464
* https://github.com/openedx/edx-platform/pull/35861
* `Private-ref`: [FAL-3943](https://tasks.opencraft.com/browse/FAL-3943)

## Testing instructions

* Checkout https://github.com/openedx/edx-platform/pull/35861
* Enable new course unit mfe page using waffle: `contentstore.new_studio_mfe.use_new_unit_page`
* Copy any text, video or problem block from v2 library and paste into any course unit.
* Update the block in library and publish the library.
* Go back to course unit when the block was pasted and use the `update` button in the component header.
* Verify that the preview changes modal should be displayed in the MFE and not inside the iframe.
* Accept or ignore changes and verify the behaviour.
